nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,8 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #44.        def _add_available_mask(data: np.ndarray, temporal_cols):
</pre><p>Please rename this to <code>_ensure_available_mask</code>, since it only adds it if it isn't already there  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>